config/initializers/session_store.rb

```diff
@@ -0,0 +1,3 @@
+# Be sure to restart your server when you modify this file.
+
+Rails.application.config.session_store :cookie_store, key: '_tada_session'
```

Comment:
@vlasiak, FYI: I would recommend to use database as storage of the session in the future. Also it is good to know the following: http://guides.rubyonrails.org/security.html